PlayCover/PlayCoverError.swift

```diff
@@ -18,15 +18,15 @@ extension PlayCoverError: LocalizedError {
         case .infoPlistNotFound:
             return NSLocalizedString("This .IPA is courrupted. It doesn't contains Info.plist.", comment: "")
         case .waitInstallation:
-            return NSLocalizedString("Please, wait for the current install to finish!", comment: "")
```

Comment:
Please revert since it seems the change is product of old code on this PR. This line will be modified by Pull request #13 for proper grammar.